Wikibase/Tests/ClassLevelDocumentationSniffTest.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Wikibase\CodeSniffer\Tests;
+
+use PHP_CodeSniffer\Config;
+use PHP_CodeSniffer\Files\DummyFile;
+use PHP_CodeSniffer\Ruleset;
+use Wikibase\Sniffs\Commenting\ClassLevelDocumentationSniff;
+
+/**
+ * @license GPL-2.0-or-later
+ */
+class ClassLevelDocumentationSniffTest extends \PHPUnit_Framework_TestCase {
```

Comment:
There is a testing framework in place in this codebase:
* The file Tests/Commenting/ClassLevelDocumentation.php lists all test cases that are relevant for a sniff.
* Tests/Commenting/ClassLevelDocumentation.php.expected contains the expected output from a `phpcs` run.
* Tests/Commenting/ClassLevelDocumentation.php.fixed contains the expected code after running `phpcbf`.

No need to repeat all the setup again.

Reply:
I know.
But there are some issues with that approach:
  1. I can't set a property for the sniff
  2. I can't test a sniff in isolation
  3. In this case (if I've managed to set the license property) I will have to fix `ClassLevelDocumentation.php.expected` and `ClassLevelDocumentation.php.fixed` although the change is backwards compatible.